### PR TITLE
Replace DynatraceMeterRegistry to JMXmeterRegistry

### PR DESCRIPTION
**Pull request by boyan-velinov**
Jira: LMCROSSITXSADEPLOY-2178

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -310,10 +310,10 @@
                 <version>${snakeyaml.version}</version>
             </dependency>
             <dependency>
-                <groupId>io.micrometer</groupId>
```
Comment (nictas) [inline pom.xml:313]: Replace the tabs with spaces.

File: multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java [hunk 2]
```diff
@@ -76,42 +49,20 @@ public Duration step() {
...
-    }
-
-    private String getDynatraceUrl(DynatraceServiceInfo serviceInfo) {
-        if (serviceInfo == null) {
-            return null;
-        }
-        return serviceInfo.getMicrometerUrl();
-    }
-
     private DynatraceServiceInfo getDynatraceServiceInfo() {
```
Comment (nictas) [inline multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java:66]: Do we need this now?

File: multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java [hunk 3]
```diff
@@ -32,41 +32,14 @@
 
     @Inject
     @Bean
-    public DynatraceMeterRegistry dynatraceMeterRegistry(ApplicationConfiguration appConfig) {
+    public JmxMeterRegistry jmxMeterRegistry(ApplicationConfiguration appConfig) {
         DynatraceServiceInfo serviceInfo = getDynatraceServiceInfo();
```
Comment (nictas) [inline multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java:36]: This should no longer be needed, right?

Comment (boyan-velinov) [reply multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java:36]: DynatraceMeterRegistry  is removed. ReactorNetty use directly Metrics.globalRegistry that propagates meters in all registered registries.

File: pom.xml (hunk 1 above)
Comment (boyan-velinov) [reply pom.xml:313]: Done

File: multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java (hunk 2 above)
Comment (boyan-velinov) [reply multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java:66]: It is not mandatory to check if DynatraceServiceInfo exists before registering JmxMeterRegistry.
However, it we check that we can optimize the things as not registering the registry at all.
In this way we will avoid unecesarry preparation of our own metrics and send them to micrometer

**Pull request merged**